tests/run-make/rustdoc-map-file/rmake.rs

```diff
@@ -0,0 +1,16 @@
+use run_make_support::{rustdoc, tmp_dir};
+use std::process::Command;
+
+fn main() {
+    let out_dir = tmp_dir().join("out");
+    rustdoc()
+        .input("foo.rs")
+        .arg("-Zunstable-options")
+        .arg("--generate-redirect-map")
+        .output(&out_dir)
+        .run();
+    let python = std::env::var("PYTHON").unwrap_or("python".into());
+    assert!(
+        Command::new(python).arg("validate_json.py").arg(&out_dir).status().unwrap().success()
+    );
```

Comment:
This is very amusing to me. Let me go check what `validate_json.py` does.

Reply:
https://github.com/rust-lang/rust/blob/e3029d220f6843f9b83d35927ff20a86137d7ed0/tests/run-make/rustdoc-map-file/validate_json.py